site/documentation/content/admin-guide/jdbc-device-registry-config.md

```diff
@@ -66,12 +66,18 @@ and availability.
 | `HONO_REGISTRY_JDBC_ADAPTER_USERNAME`            <br> `hono.registry.jdbc.adapter.username`             | no  | -    | The username used to access the database. |
 | `HONO_REGISTRY_JDBC_ADAPTER_PASSWORD`            <br> `hono.registry.jdbc.adapter.password`             | no  | -    | The password used to access the database. |
 | `HONO_REGISTRY_JDBC_ADAPTER_MAXIMUMPOOLSIZE`     <br> `hono.registry.jdbc.adapter.maximumPoolSize`      | no  | Depends on the connection pool implementation. `15` for C3P0. | The maximum size of the connection pool. |
+| `HONO_REGISTRY_JDBC_ADAPTER_MINIMUMPOOLSIZE`     <br> `hono.registry.jdbc.adapter.minimumPoolSize`      | no  | Depends on the connection pool implementation. `3` for C3P0.   | The minimum size of the connection pool. |
```

Comment:
since there is no way for a user to configure the particular connection pool implementation to be used, I believe that we should remove the reference to `C3P0` here (and in the other places). I would also propose to define reasonable default values for the arbitrary properties in the `JdbcOptions` class itself, instead of relying on the implicit defaults of C3P0 ...

Reply:
I removed the references and defined default values in the `JdbcOptions` class.